examples/lsm303dlh_mag_tiltcompensatedcompass.py

```diff
@@ -0,0 +1,86 @@
+# Tilt Compensated Compass using Accelerometer and Magnetometer
+# SPDX-FileCopyrightText: 2021 Alex Dew
+# SPDX-License-Identifier: MIT
+
+# The equations below are the ones used in code
+# Equations from st manual:
+# "Computing tilt measurement and tilt-compensated eCompass" by Andrea Vitali
+# https://www.st.com/resource/en/design_tip/dm00269987-computing-tilt-measurement-and-tiltcompensated-ecompass-stmicroelectronics.pdf
+
+# for Phi (Roll)
+# Phi = atan2(Gy, Gz)
+# for stability, 'Gz' can be substituted with Gz+Gx*alpha (alpha=0.01-0.05)
+
+# for Theta (Pitch)
+# Gz2 = Gy * Sin( Phi ) + Gz * Cos( Phi )
+# Theta = Atan(-Gx / Gz2)
+
+# for Psi or "Yaw" (The Tilt Compensated Heading)
+# Bx,By,Bz are magnetometer readings
+
+# By2 = Bz * Sin( Phi ) – By * Cos( Phi )
+# Bz2 = By * Sin( Phi ) + Bz * Cos( Phi )
+# Bx3 = Bx * Cos( Theta ) + Bz2 * Sin( Theta )
+# Psi = Atan2(By2 , Bx3) # This is the Yaw (Tilt Compensated Heading)
+###############################################
+
+import board
+import busio
+from digitalio import DigitalInOut
+import re
+import time
+from math import atan, atan2, degrees, pi, cos, sin
+import adafruit_lsm303_accel
+import adafruit_lis2mdl
+import adafruit_lsm303dlh_mag
+
+# Mag and Accel offsets from calibration. You need to calibrate your own offset
+offmag = (-10.8, 4.875, -54.225)
+offaccel = (-0.994392, -0.305969, 0.382458)
+
+
+i2c = board.I2C()  # uses board.SCL and board.SDA
+sensoraccel = adafruit_lsm303_accel.LSM303_Accel(i2c)
+sensormag = adafruit_lis2mdl.LIS2MDL(i2c)
```

Comment:
I am guessing that here, we would need to change to use the magnetic information for the sensor in this library. So related to previous comment

Reply:
When I do that, the values are wrong.  ...I'll look more if I have time this afternoon.  Maybe for this library, the values just need to be swapped around somehow.

Reply:
Interesting, I have both sensors so maybe we can compare results to see what is hapenning

Reply:
Awesome.  That would be great if you could check and then we can include the example in both places.

Reply:
No problem.. will do this weekend :)